Parity signer page and MyCrypto tutorial

Tbaut opened the pull request:
Fix https://github.com/paritytech/parity-signer/issues/88

5chdn commented on Parity-Signer-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,78 @@
+You will now be presented with a QR quode that represents the transaction from your phone account, to the adress entered in the previous step on MyCrypto. Now obviously only the account that sits on your phone can sign and authorize this transaction. This is what we'll do.
+- From the Parity Signer app, click on `Scan transaction QR` and scann the QR code presented by MyCrypto
+- Review the transaction addresses and the amount to send
+- I repeat - make sure everything matches, if you got pished, this is where you can realize !
```
~~I repeat - m~~ Make sure everything matches ~~,~~ if you got _phished_, this is where you can realize _and reject the transaction!_

Tbaut replied:
fixed

5chdn commented on Parity-Signer-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,78 @@
+
+![create account](images/Parity-Signer-android-1.png)
+
+Your phone has now signed the transaction offline using your Parity Signer account private key. The QR code that is now on your phone represents a signed transaction that can be broadcasted. Will will do this in the next steps.
```
Make **signed offline** bold, this is such an incredible key feature :)

5chdn commented on Parity-Signer-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,78 @@
+- Show your phone to your webcam for MyCrypto to be able to read the signed transaction
+- Verify one last time that the account addresses are correct as well as the amount to transfer
+- Click on `send` on MyCrypto
+- BOOM you just sent Ether from an air-gapped wallet !
```
~~Boom~~ ? :)

No space between word and exclamation mark ;)

Also tell users to try with small amounts first :)

5chdn commented on Parity-Signer-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,78 @@
+- BOOM you just sent Ether from an air-gapped wallet !
+
+![Mycrpto parity signer app](images/MyCrypto-Parity-Signer-4.jpg)
+![Mycrpto parity signer app](images/MyCrypto-Parity-Signer-5.jpg)
```
I think the screenshots are in wrong order, could you double-check that?

5chdn commented:
See https://wiki.parity.io/JSONRPC-signer-module.html and https://wiki.parity.io/Parity-Wallet.html and make sure we do not confuse the signer with the signer.

Ref. https://github.com/MyCryptoHQ/MyCrypto/pull/1349

5chdn commented:
Also adding on-ice label because we should not publish this before it's actually available in mycrypto.

maciejhirsz commented:
IDK about including "Mobile" in the name since the app is called "Parity Signer" in the app store and google play.

Tbaut commented on Parity-Signer.md:
```diff
@@ -0,0 +1,15 @@
+# Parity Signer App
```
fixed

Tbaut commented:
- I've used the name *Parity Signer Mobile App* across all tutorials.
- Added a very details Step by Step tuto to use Parity Signer Mobile App together with Parity Wallet
- Added some gifs to show some steps, hope you like it ;)

dternyak commented:
I just wanted to pop-in and say a big thank you for creating these tutorials! :) 

This will be very helpful for new users who are trying to get familiar with the various wallet formats and the many advantages of using Parity Signer as their private key manager.

maciejhirsz commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,80 @@
+  - [Recover an account with your recovery phrase](#recover-an-account)
+- [3. Sign a transaction on MyCrypto](#3-sign-a-transaction-on-mycrypto)
+
+<br/>
```
These seem to ruin formatting on github for whatever reason. I think it's okay to remove them, the default spacing before a headline should be fine.

Tbaut replied:
fixed

maciejhirsz commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,80 @@
+<br/>
+## 3. Sign a transaction on MyCrypto
+
+To be able to follow this tutorial and send Ethers from this freshly created account on Parity Signer Mobile App, you will obviously need to get some Ethers to send. Use the account address or the QR code available on your account main view to get Ethers. Follow the steps 3. and 4. of the [Parity Wallet tutorial](Parity-Signer-Mobile-App-Parity-Wallet-tutorial#3-attach-a-parity-signer-mobile-app- ...
```
AFAIK "Ether" is a non-enumerable noun (like "water" or "money"), so all "Ethers" should be changed to "Ether".

Tbaut replied:
Thanks, fixed

maciejhirsz commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,80 @@
+
+![Mycrpto parity mobile signer app](images/MyCrypto-Parity-Signer-1.jpg)
+
+On your phone, select the account you want to send Ethers from and make sure that the QR is fully displayed on the phone before facing your phone's screen to your webcam.
```
> (...) select the account you want to send Ethers from (...)

This is actually not necessary, the QR code contains the address from which you want to send Ether and the mobile app will prompt up with the correct account if it exists, it should error otherwise.

Tbaut replied:
You need to select an account to get a QR code displayed on your phone. I agree that you do not need to select an account to click on `Scan transaction QR` to sign a transaction (this is somewhere else in the tuto, I'll correct that ;) ) but this part is necessary to let MyCrypto select the right account.

maciejhirsz commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,80 @@
+You will now be presented with a QR quode that represents the transaction. Since this transaction is sending funds from your Parity Signer Mobile App account, only this account (siting on your phone) is able sign and authorize this transaction. This is what we'll do in the next steps:
+- From the Parity Signer Mobile App, click on `Scan transaction QR` and scan the QR code presented by MyCrypto.
+- Review the transaction addresses and the amount to send.
+- Make sure everything matches, if you got phished, this is where you can realize it and reject the transaction!
```
I'd combine this point and the one above into a single one:

- Review the transaction as a phishing precaution. The amount and addresses must match what you've entered in MyCrypto.

Tbaut replied:
fixed

Tbaut replied:
I really want to insist here, this costs a couple more words but people should really check carefully.

Tbaut commented:
Grumble addressed ;)

kirushik commented on Parity-Signer-Mobile-App.md:
```diff
@@ -0,0 +1,21 @@
+## What is it ?
+Parity Signer Mobile App is a secure air-gapped wallet that allows users to use a smartphone as cold storage. It means that, once the App has been installed, users can create Ethereum accounts, sign transactions and [transfer funds](Parity-Signer-Mobile-App-Parity-Wallet-tutorial) from and to these accounts without any sort of connectivity enabled on the device. Wifi, Mobile Network, Bluetooth ca ...
+
+Any data transfer from or to the App will happen using QR code scanning. By doing so, the most sensitive piece of information, the private keys, will never leave the phone. The Parity Signer Mobile App can be used to store any Ethereum account, this includes ETH, ETC as well as Ether from various testnets (Kovan, Ropsten...).
```
@Tbaut will it be too much to ask to address [parity-signer/issues#89](https://github.com/paritytech/parity-signer/issues/89) here? It looks like the best place.

Basically it should recommend:
 - getting a separate device for signer, 
 - factory-resetting it (just in case),
 - enabling full-disk encryption on the device, with reasonable password (might not be on by default, for example for older Android devices);
 - definitely not using any kind biometrics for device decryption/unlocking;
 - enabling the plane mode on the device forever after the Parity Signer is installed.

Tbaut replied:
@kirushik Sure no problem, what about never connecting it to the internet and install the signed APK directly with usb connection for Android?
About updates, what should we advise ?

kirushik replied:
> install the signed APK directly with usb connection for Android

I personally don't see that as a valuable measure — it would reduce one's exposure to Google services (good), but still require enabling the "Install Unauthorized apps" checkbox in settings (bad). I don't think brings in any significant security improvements, while being unnecessary complicated.
We should consider uploading Signer to F-Droid at some point, though.

> About updates, what should we advise?

I would suggest asking for the wider set of opinions in Riot here; but my position is that it should be completely safe to never connect to the network and never install any updates.

Tbaut replied:
Added the suggestions from your 1st comment

maciejhirsz commented:
I'd suggest we deploy this for MyCrypto beta, and tackle #89 in a separate PR.

Tbaut commented:
Just addressed the comments, it should be good to go ;)

Tbaut commented on Parity-Signer.md:
```diff
@@ -0,0 +1,15 @@
+- [iOs](https://itunes.apple.com/us/app/parity-signer/id1218174838)
+
+Please double check carefully the origin of the app, and that the company distributing it is **Parity Technologies** 
+Follow the [tutorial with MyCrypto](Parity-Signer-MyCrypto-tutorial) that will walk you through the creation of an account and show you how to send funds out of an account using MyCrypto.com.
```
fixed

Tbaut commented on Parity-Signer.md:
```diff
@@ -0,0 +1,15 @@
+Follow the [tutorial with MyCrypto](Parity-Signer-MyCrypto-tutorial) that will walk you through the creation of an account and show you how to send funds out of an account using MyCrypto.com.
+
+## What data does it collect ?
+None, it's as simple as this, Parity Signer Android and iOs does not send any sort of data to Parity Technologies or any partner.
```
fixed

axelchalon commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,79 @@
+
+![create account](images/Parity-Signer-android-1.png)
+
+Your phone has now *signed the transaction offline* using your Parity Signer Mobile App account private key. The QR code that is now displayed your phone represents a signed transaction that can be broadcasted. We will do this in the next steps:
```
you forgot a word there ^^
displayed on your phone*

axelchalon commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,79 @@
+
+![Mycrpto parity mobile signer app](images/MyCrypto-Parity-Signer-3.jpg)
+
+You will now be presented with a QR quode that represents the transaction. Since this transaction is sending funds from your Parity Signer Mobile App account, only this account (siting on your phone) is able sign and authorize this transaction. This is what we'll do in the next steps:
```
sitting on your phone

axelchalon replied:
QR code

axelchalon commented on Parity-Signer-Mobile-App-Parity-Wallet-tutorial.md:
```diff
@@ -0,0 +1,109 @@
+
+
+## 2. Setup or recover an account
+When launching the app for the first time, no account has been set up yet. At this stage, you will either want to create an account directly from your mobile device or recover an account already created in with Parity Signer Mobile App or [Parity Wallet](https://wiki.parity.io/Parity-Wallet) (Desktop)
```
>in with

axelchalon commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,79 @@
+
+To be able to follow this tutorial and send Ether from this freshly created account on Parity Signer Mobile App, you will obviously need to get some Ether to send. Use the account address or the QR code available on your account main view to get Ether. Follow the steps 3. and 4. of the [Parity Wallet tutorial](Parity-Signer-Mobile-App-Parity-Wallet-tutorial#3-attach-a-parity-signer-mobile-app-acc ...
+
+Assuming that your account now has funds, you will be able to send securely some Ether to anyone, without transfering your private key, and without needing any internet connection on your mobile phone.
```
transferring

axelchalon commented on Parity-Signer-Mobile-App-MyCrypto-tutorial.md:
```diff
@@ -0,0 +1,79 @@
+
+To do so, visit [https://mycrypto.com/](https://mycrypto.com/), click on `Parity Signer` and allow the website to access your webcam. This is required to scan the QR code from your phone.
+
+![Mycrpto parity mobile signer app](images/MyCrypto-Parity-Signer-1.jpg)
```
typo on mycrypto for this alt and the other ones too

5chdn commented:
:+1: Grammarly :+1:

Tbaut commented:
Thanks, @axelchalon.
I fixed them here : https://github.com/paritytech/wiki/pull/65

revolution001 commented:
just bug all fcuking saying me similiar..i can destory this etheruem
